README.md

```diff
@@ -2,24 +2,29 @@
 
 [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://apps.vechain.org/)
 
-App-Hub is a platform that makes your VeChain App easily discovered worldwide. Once the pull request has been merged, your app will be shown in the App-Hub.
+First off, thanks for making the awesome app on VeChain and taking the time to contribute 💪
 
-## Requirements :
-1. The app must run in Mainnet
+The following is a set of guidelines for contributing to **AppHub**. These
+are just guidelines, not rules. Use your best judgment and feel free to propose changes to this document in a pull request.
+
+## Rules
+1. The app must run on Mainnet
 2. Support [Connex](https://connex.vecha.in/#/)
 3. Logo is required 
 4. The id must be unique 
-5. Clear description 
+5. Short and simple descriptions 
 6. Comply with directory & contents rules
-7. One application per submission 
+7. One application per submission
+
+## Adding your app
+
+[Fork and clone this repository](https://help.github.com/articles/fork-a-repo/) to create your app details and make a pull request for the maintainers to review your app and  merge into AppHub.
```

Comment:
It may be overkill for details, but the inclusion of "fork repo + create a change-specific branch" may be helpful in reducing the multi-PR/unclean history issues we've seen. Including that step for first time Git users (as I was) would have saved everyone from some headaches.

Reply:
good suggestion! i will change that .